### PR TITLE
simple-singolo

### PR DESCRIPTION
**Pull request by Pishuhott**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+function shuffle(array) {
+	array.sort(() => Math.random() - 0.5);
+};
+
+
+//=====Slider======
+
+(function() {
+
+    var doc = document,
```
Comment (castortroy06) [inline js/main.js:88]: what did it give you? :)

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        index = 1;
+
+    var Slider = function() {
```
Comment (castortroy06) [inline js/main.js:91]: 1. Is it your code below?
2. if yes, why did you add CSS to the JS script?

File: sass/abstract/_constants.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+//переменные
```
Comment (castortroy06) [inline sass/abstract/_constants.scss:1]: Please, don't use Russian comments.

File: sass/abstract/_constants.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+$color-header: #2D303A;
+$color-services: #F2F2F2;
+$color-background: #2D303A;
+
+
+
+$content-width: 1020px;
+$tablet-width: 768px;
+$mobile-width: 420px;
+$mobile-small-width: 360px;
```
Comment (castortroy06) [inline sass/abstract/_constants.scss:17]: Don't forget to add an empty line at the end of each file

File: sass/base/_base.scss [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    list-style-type: none;
+    text-align: justify;
+}
+
+
+h1 {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 25px;
+    line-height: 30px;
```
Comment (castortroy06) [inline sass/base/_base.scss:18]: Please, use relative value for the line-height. 
`line-height: 1.2;`

File: sass/base/_base.scss [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+h2 {
+    color: $color-heading;
+    font-style: normal;
+    font-weight: 900;
+    font-size: 30px;
+    line-height: 36px;
```
Comment (castortroy06) [inline sass/base/_base.scss:26]: and here

Comment (castortroy06) [reply sass/base/_base.scss:26]: and further

File: sass/base/_media.scss [hunk 7]
```diff
@@ -0,0 +1,232 @@
+@media (max-width: $tablet-width) {
```
Comment (castortroy06) [inline sass/base/_media.scss:1]: please, use code-formatter for the JS and SCSS if possible.

File: sass/base/_media.scss [hunk 8]
```diff
@@ -0,0 +1,232 @@
...
+        }
+        border-bottom: 7px solid #323746
+    }
+}
+
+@media (max-width: 576px) {
+    .header {
+        max-height: 71px;
+        border: none;
+        & .header--fixed {
```
Comment (castortroy06) [inline sass/base/_media.scss:59]: you no need to use "& ", you can just remove it. you have to use & for concatenation of classes like
```
.header {
  &.header-fixed
}   ------->>>>   .header.header-fixed
```

File: sass/component/_burger.scss [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+    height: 100%;
+    background-color: rgba($color: $color-background, $alpha: .5);
+    transition: .5s;
+}
+
+a.navigation__link-active {
+    color: $color-primary;
+}
+
+
```
Comment (castortroy06) [inline sass/component/_burger.scss:80]: ||
||
||
||
||
||
||
||
\\/


????

File: sass/layouts/_portfolio.scss [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+    &:hover {
+        color: white;
+        border: 1px solid white;
+    }
+    
+}
+.portfolio-tab-list .active-tab {
+    color: white ;
+    border: 1px solid white;
+}
```
Comment (castortroy06) [inline sass/layouts/_portfolio.scss:69]: empty line

File: sass/layouts/_wrapper.scss [hunk 11]
```diff
@@ -0,0 +1,4 @@
+.wrapper {
+    max-width: 1020px;
+    margin: auto;
+}
```
Comment (castortroy06) [inline sass/layouts/_wrapper.scss:4]: empty line

File: sass/main.scss [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+@import 'component/slider';
+@import 'component/burger';
+
+@import 'layouts/wrapper';
+@import 'layouts/header';
+@import 'layouts/services';
+@import 'layouts/portfolio';
+@import 'layouts/footer';
+
+@import 'base/media';
```
Comment (castortroy06) [inline sass/main.scss:20]: empty line

**Review by castortroy06 — COMMENTED** [review]
please, review comments

File: sass/main.scss (hunk 12 above)
Comment (Pishuhott) [reply sass/main.scss]: corrected comments

File: sass/layouts/_wrapper.scss (hunk 11 above)
Comment (Pishuhott) [reply sass/layouts/_wrapper.scss]: corrected comments

File: sass/layouts/_portfolio.scss (hunk 10 above)
Comment (Pishuhott) [reply sass/layouts/_portfolio.scss]: corrected comments

File: sass/component/_burger.scss (hunk 9 above)
Comment (Pishuhott) [reply sass/component/_burger.scss]: corrected comments

File: sass/base/_media.scss (hunk 8 above)
Comment (Pishuhott) [reply sass/base/_media.scss]: corrected comments

File: index.html [hunk 13]
```diff
@@ -0,0 +1,271 @@
...
+        <div class="burger-menu__overley"></div>
+    </div>
+
+    <main>
+        <section class="slider">
+            <div class="wrapper">
+                <div class="swiper-container">
+                    <div class="swiper-elements">
+                        <img id="lastClone" src="img/slider/Slide-2.svg" alt="" class="slide">
+                        <img src="img/slider/Slide-1.svg" alt="" class="slide">
```
[the remark this answers is not in the document]
Comment (Pishuhott) [reply index.html]: corrected comments

File: index.html [hunk 14]
```diff
@@ -0,0 +1,271 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/style.css">
+    <title>SINGLO</title>
+</head>
+
+<body>
+    <header id="haeder-section" class="header">
```
[the remark this answers is not in the document]
Comment (Pishuhott) [reply index.html]: corrected comments

File: index.html [hunk 15]
```diff
@@ -0,0 +1,271 @@
...
+        <div class="header--fixed">
+            <div class="wrapper">
+                <div class="logo">
+                    <a href="#">
```
[the remark this answers is not in the document]
Comment (Pishuhott) [reply index.html]: corrected comments

File: index.html [hunk 16]
```diff
@@ -0,0 +1,271 @@
...
+                    <li class="social__link">
+                        <a href="#">
+                            <img src="img/footer/ico-linkedin.svg" alt="">
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+    </footer>
+    <script src="js/main.js"></script>
```
[the remark this answers is not in the document]
Comment (Pishuhott) [reply index.html]: corrected comments

File: js/main.js [hunk 17]
```diff
@@ -0,0 +1,150 @@
+//=====SCROLL=====
```
[the remark this answers is not in the document]
Comment (Pishuhott) [reply js/main.js]: corrected comments

File: sass/abstract/_constants.scss (hunk 3 above)
Comment (Pishuhott) [reply sass/abstract/_constants.scss:1]: corrected comments

File: sass/base/_base.scss (hunk 5 above)
Comment (Pishuhott) [reply sass/base/_base.scss:18]: in the future I will consider

File: sass/base/_base.scss (hunk 6 above)
Comment (Pishuhott) [reply sass/base/_base.scss:26]: in the future I will consider

File: sass/base/_media.scss (hunk 7 above)
Comment (Pishuhott) [reply sass/base/_media.scss:1]: in the future I will consider

**Comment by castortroy06** [comment]
@Pishuhott please resolve all conversations and conflicts
